### DISCUSSION TITLE
(remark-gfm) +1 on shipping ESM-only

### DISCUSSION THREAD

**Discussion by ctjlewis** [remarkjs/remark#1117] [General]
Thank you for your service. From Next.js docs.

<img width="615" alt="image" src="https://user-images.githubusercontent.com/1657236/220287036-7c705749-ee0d-410e-bc84-80c3682cb89a.png">

**Comment by Murderlon** [comment]
Hi, do you have a question? This seems more like a statement.

**Reply by wooorm** [reply]
I think it’s just someone who’s also a fan of ESM saying thanks! :)

Glad you like it @ctjlewis!
